src/main/java/racingcar/domain/Car.java

```diff
@@ -0,0 +1,64 @@
+package racingcar.domain;
+
+
+import java.util.Objects;
+
+public class Car {
+
+    private Position position;
+    private Name name;
+    private MoveStrategy moveStrategy;
+
+    public Car(String name, MoveStrategy moveStrategy) {
+        this.name = new Name(name);
```

Comment:
```suggestion
        this(name, new Position(0), moveStrategy);
```

부생성자는 주생성자(인스턴스 변수와 인자가 같은 생성자) 를 호출하도록 해보시면 좋을것 같아요!
하단생성자를 주생성자로 만들기위해 Name 또한 String 이 아닌 Name 타입으로 받아볼수 있겠네요!